src/app/digital/views/components/BUFGateView.ts

```diff
@@ -0,0 +1,16 @@
+import {DEFAULT_BORDER_COLOR, DEFAULT_BORDER_WIDTH, DEFAULT_CURVE_BORDER_WIDTH, SELECTED_BORDER_COLOR} from "core/utils/Constants";
+
+import {V} from "Vector";
+
+import {BUFGate} from "core/models/types/digital";
+
+import {ComponentView} from "core/views/ComponentView";
+
+import {DigitalViewInfo} from "../DigitalViewInfo";
+
+
+export class BUFGateView extends ComponentView<BUFGate, DigitalViewInfo> {
+    public constructor(circuit: DigitalViewInfo, obj: BUFGate) {
+        super(circuit, obj, V(1, 1), "buf.svg");
+    }
+}
```

Comment:
You don't need a custom view here, you can just do `new ComponentView<BUFGate>(c, o, V(1, 1), "buf.svg")` in `src/app/digital/views/index.ts`
(since all this class does is call the `super` constructor which is `ComponentView` and has no extended functionality)